docusaurus/docs/React/utility-components/channel-preview-ui.mdx

```diff
@@ -0,0 +1,203 @@
+<ChannelList
+  Preview={(previewProps) => <YourCustomChannelPreview {...previewProps} />}
+/>;
+```
+
+## Props
+
+### <div class="label required basic">Required</div> channel
+
+This required prop is the channel to be previewed; comes from either the `channelRenderFilterFn` or `usePaginatedChannels` call from
+[ChannelList](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelList/ChannelList.tsx) and does not need to be set manually.
+
+| Type    |
+| ------- |
+| Channel |
+
+### active
+
+If the component's channel is the active (selected) channel.
+
+| Type    |
+| ------- |
+| boolean |
+
+### activeChannel
+
+The currently selected channel object.
+
+| Type    |
+| ------- |
+| Channel |
+
+### Avatar
+
+The custom UI component to display the an avatar for the channel.
+
+| Type      | Default                                                                                               |
+| --------- | ----------------------------------------------------------------------------------------------------- |
+| component | [Avatar](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Avatar/Avatar.tsx) |
+
+### channelUpdateCount
+
+A number that forces the update of the preview component on channel update.
+
+| Type   |
+| ------ |
+| number |
+
+
+### displayImage
+
+Image of channel to display.
+
+| Type   |
+| ------ |
+| string |
+
+
+### displayTitle
+Title of channel to display.
+
+| Type   |
+| ------ |
+| string |
+
+### lastMessage
+The last message received in a channel.
+
+| Type          |
+| ------------- |
+| StreamMessage |
+
+### latestMessage
+Latest message preview text to display.
+
+| Type   |
+| ------ |
+| string |
+
+### latestMessageLength
+
+Text truncation limit for latest message.
+
+| Type   |
+| ------ |
+| number |
+
+### Preview
+
+The UI component to use for display.
+
+| Type      | Default                                                                                                                                         |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| component | [ChannelPreviewMessenger](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelPreview/ChannelPreviewMessenger.tsx) |
+
+### setActiveChannel
+
+The setter function for a selected `channel`.
+
+| Type                                 |
+| ------------------------------------ |
+| ChatContextValue['setActiveChannel'] |
+
+### unread
+The number of unread Messages.
+
+| Type   |
+| ------ |
+| number |
+
+### watchers
+
+The object containing watcher parameters. See the [Pagination documentation](https://getstream.io/chat/docs/react/channel_pagination/?language=js) for a list of available fields for sort.
+
+| Type                                |
+| ----------------------------------- |
+| { offset?: number; limit?: number } |
+
+### active
```

Comment:
looks like they're duplicates from here on down

Reply:
eeek, really bad copy and paste I think.